letsencrypt/cli.py

```diff
@@ -695,6 +730,116 @@ def install(config, plugins):
     le_client.enhance_config(domains, config)
 
 
+def renew(cli_config, plugins):
+    """Renew previously-obtained certificates."""
+    cli_config = configuration.RenewerConfiguration(cli_config)
+    if cli_config.domains != []:
```

Comment:
This turns out to make it impossible to pass `--webroot-map` to the renewer for the purposes of changing the directories of webroot entries, which is a pity.